rpcs3/Crypto/unself.cpp

```diff
@@ -1396,7 +1397,7 @@ bool SELFDecrypter::GetKeyFromRap(u8* content_id, u8* npdrm_key)
 
 	// Try to find a matching RAP file under exdata folder.
 	const std::string ci_str = reinterpret_cast<const char*>(content_id);
-	const std::string rap_path = "/dev_hdd0/home/00000001/exdata/" + ci_str + ".rap";
+	const std::string rap_path = fmt::format("/dev_hdd0/home/%s/exdata/%s.rap", Emu.GetUsr(), ci_str);
```

Comment:
According to KD-11's comment you should use:
`"/dev_hdd0/home/" + Emu.GetUsr() + "/exdata/" + ci_str + ".rap"`